### DISCUSSION TITLE
Generate map content only inside borders

### DISCUSSION THREAD

**Discussion by tisis2** [openmaptiles/openmaptiles#1441] [Q&A]
Hi there,

i was wondering if it would be possible with openmaptiles to generate maps without having data outside the borders. when i generate maps with default configuration the selected area always has some landuse and sead etc around the border. if it is possible i would like to be able lo generate maps that are ending exactly at the border so it would easily be possible to layer maps to combine different areas in a mapviewer.
maybe there is a solution and i am just missing it... thanks for your help or any ideas about it.

**Comment by TomPohys** [comment]
To get a clear cut in mbtiles, I think that the best approach will be cut all geometries (with the boundary (admin_level=2)) in tables which are input into the final function (e.g. `layer_landuse`).  After the `import-sql` step and delete what is not within this boundary.

**Reply by tisis2** [reply]
how difficult would that be to do? i totally not know anything about postgreSQL yet... until now i was mainly using the quickstart with some custom configurations. maybe someone can help me with a reference or small example how this approach could look

**Reply by andreassteffens** [reply]
Wondering if you ever figured out a nice and tidy way to do this?

**Reply by tisis2** [reply]
no sry. i did not dive that deep into that topic... and since i do not need it anymore i wont look in to it deeper anymore
